### PR TITLE
UI: landing page of pure lander and ui setup

### PR DESCRIPTION
**Pull request by Sonika-dot-HP**
- landing page of pure lander and ui setup

### CODE REVIEW

File: src/shared/services/util.service.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
+/*
+*  Copyright 2018-19, MapleLabs, All Rights Reserved.
+*/
+
+
```
Comment (Ganesh-dot-Kaushik) [inline src/shared/services/util.service.ts:5]: are we using this service

File: src/shared/services/http/active-xhrs-list.services.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { Injectable, EventEmitter } from '@angular/core';
+// import { Observable } from 'rxjs/Observable';
```
Comment (Ganesh-dot-Kaushik) [inline src/shared/services/http/active-xhrs-list.services.ts:2]: remove the comments

File: src/shared/services/application-data.service.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
+/*
+*  Copyright 2018-19, MapleLabs, All Rights Reserved.
+*/
+
```
Comment (Ganesh-dot-Kaushik) [inline src/shared/services/application-data.service.ts:4]: if not using the file remove

**Review by Ganesh-dot-Kaushik — CHANGES_REQUESTED** [review]

File: src/shared/services/util.service.ts (hunk 1 above)
Comment (Sonika-dot-HP) [reply src/shared/services/util.service.ts]: No, im not using it
i have removed the file

File: src/shared/services/http/active-xhrs-list.services.ts (hunk 2 above)
Comment (Sonika-dot-HP) [reply src/shared/services/http/active-xhrs-list.services.ts]: okay , i will remove it

File: src/shared/services/application-data.service.ts (hunk 3 above)
Comment (Sonika-dot-HP) [reply src/shared/services/application-data.service.ts]: the file is interlinked and cannot be removed.

**Pull request merged**